src/app/client/src/app/modules/resource/components/collection-player/collection-player.component.ts

```diff
@@ -142,10 +142,23 @@ export class CollectionPlayerComponent implements OnInit, OnDestroy {
         id: content.metadata.identifier,
         type: content.metadata.contentType || content.metadata.resourceType || 'content',
         ver: content.metadata.pkgVersion || '1',
-        rollup: {l1: this.collectionId}
+        rollup: { l1: this.collectionId }
         // rollup: this.collectionInteractObject
       };
       this.triggerContentImpression = true;
+      // if mimeType is text/x-url extcontentpreview plugin will be invoked
+      if (content.metadata.mimeType === 'text/x-url') {
+        let msg = 'As the content is from an external source, it will be opened in a new tab.';
```

Comment:
we should not have hardcoded messages. all texts should come from resource bundles.

Reply:
Since yesterday, resourceServices is not working, so we were getting error, if resourceService is working fine, the hardcoded message will be removed

Reply:
done